src/jit/lsra.cpp

```diff
@@ -1868,6 +1868,14 @@ void LinearScan::identifyCandidates()
             continue;
         }
 
+#if FEATURE_MULTIREG_ARGS_OR_RET        
+        if (varDsc->lvIsMultiRegArgOrRet)
+        {
+            varDsc->lvLRACandidate = 0;
+            continue;
+        }        
+#endif
+
```

Comment:
I think that a better way to deal with this is to always set `lvDoNotEnregister` when `lvIsMultiRegArgOrRet` is set, but that is a larger change, and since that code will change in the near future, this seemed a better (smaller scoped) fix.